### PR TITLE
Remove ad hoc flycheck warnings

### PR DESCRIPTION
**Pull request by matsl**
# What

Remove ad hoc flycheck warnings.

# Why

Just started the day with doing some cleanup of various flycheck warnings ranging from adding or fixing comments, removing white spaces...

### CODE REVIEW

File: hypb.el [hunk 1]
```diff
@@ -379,9 +379,11 @@ Use optional OUT-BUF if present, else the current buffer."
       (load "hbut.el"))
   (setq debug-on-error t))
 
-;; Copied from eww.el so as to not require that package.
-(defun hypb:decode-url (string)
-  (let* ((binary (url-unhex-string string))
+;; Copied from eww.el, eww-decode-url-file-name, so as to not require
+;; that package.
+(defun hypb:decode-url (url-file-name)
+  "Decode an URL-FILE-NAME."
```
Comment (rswgnu) [inline hypb.el:385]: a instead of an

File: hsys-org.el [hunk 2]
```diff
@@ -211,6 +213,7 @@ Return t if Org is reloaded, else nil."
 	     t)))))
 
 (defun hsys-org-get-libraries-to-reload ()
+  "Return all org libraries that needs to be reloaded to avoid mixed versions."
```
Comment (rswgnu) [inline hsys-org.el:216]: need instead of needs

**Review by rswgnu — CHANGES_REQUESTED** [review]
Just two grammar issues.

**Comment by matsl** [comment]
@rswgnu Fixed review comments in [6a39b75](https://github.com/rswgnu/hyperbole/pull/468/commits/6a39b756eae5457c9b161f5031eebb9a6eca4c4c) and rebased. PTAL

**Review by rswgnu — APPROVED** [review]

**Pull request merged**